asar-teensy-firmware/TestScripts/XBEE/XBEE_classTest/XBEE.h

```diff
@@ -0,0 +1,43 @@
+/* XBEE.h
+ *  Author: Aaron Mallabar
+ *  Date:   March 6,2018
+ *  Description: Header file that wraps us the serial commands to read info from the XBEE
+ *				and interprets the information into executable directions
+ *  
+ */
+
+#ifndef XBEE_H
+#define XBEE_H
+
+namespace ASAR
+{
+	class XBEE
+	{
+		public:
+			XBEE();		//Constructor (Should probably take in a baud rate)
+			~XBEE();	//Desturctor
+
+			/*Set of arrays that include all the instrctions  *
+			 *(index of array is instruction number           */
+			int action[20] = {0x00};
```

Comment:
Make these private, otherwise other classes can get to your variables and modify them.

Reply:
I don't know if this will work, because I want to read these variable from outside the class, I guess I can probably use a reference? Unchanged for now